### PR TITLE
add kitronik access bit

### PR DESCRIPTION
**Pull request by microbit-mark**
add ACCESS:bit cc @KitronikLtd

### CODE REVIEW

File: docs/extensions.md [hunk 1]
```diff
@@ -87,6 +87,10 @@ Check out [microbit.org](http://microbit.org/resellers/) for more information on
   "name": "Inex iKB-1 controller board",
   "url":"/pkg/jcubuntu/pxt-iKB1",
   "cardType": "package"
+}, {
+  "name": "Kitronik ACCESS:bit",
+  "url":"/pkg/pxt-kitronik-accessbit",
```
Comment (xuefengedu) [inline docs/extensions.md]: `/pkg/KitronikLtd/pxt-kitronik-accessbit`

File: targetconfig.json [hunk 2]
```diff
@@ -111,7 +111,8 @@
             "alankrantas/pxt-MAX7219_8x8",
             "KitronikLtd/pxt-kitronik-zip-tile",
             "lwchkg/pxt-proportional-font",
-            "jcubuntu/pxt-iKB1"
+            "jcubuntu/pxt-iKB1",
+            "pxt-kitronik-accessbit"
```
Comment (xuefengedu) [inline targetconfig.json]: `KitronikLtd/pxt-kitronik-accessbit`

**Comment by abchatra** [comment]
@microbit-mark  Please resolve the comments

File: docs/extensions.md (hunk 1 above)
Comment (microbit-mark) [reply docs/extensions.md]: Ack. Fixed

File: targetconfig.json (hunk 2 above)
Comment (microbit-mark) [reply targetconfig.json]: Ack. Fixed

**Pull request merged**